crates/matrix-sdk-crypto/src/store/memorystore.rs

```diff
@@ -63,6 +63,8 @@ pub struct MemoryStore {
     direct_withheld_info: Arc<DashMap<OwnedRoomId, DashMap<String, RoomKeyWithheldEvent>>>,
     custom_values: Arc<DashMap<String, Vec<u8>>>,
     leases: Arc<DashMap<String, (String, Instant)>>,
+    secret_inbox: Arc<DashMap<String, Vec<GossippedSecret>>>,
+    backup_keys: Arc<RwLock<BackupKeys>>,
```

Comment:
I wonder if we need all these `Arc`s. The `MemoryStore` currently implements `Clone`, but does it have to? It's not a requirement for the `CryptoStore` trait.

Reply:
Seems like the answer is no, since `Store` puts the `CryptoStore` behind an `Arc`. That's probably something left over from before we had the `Store` struct.